src/openqaoa-pyquil/requirements.txt

```diff
@@ -1 +0,0 @@
-pyquil
```

Comment:
Why was pyquil removed?

Reply:
I removed the requirements.txt but added the dependency in the pyproject